news/2017-07-03-scorewatch-june-week-4.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Scorewatch: June Week 4"
+permalink: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+date: 2017-07-03 02:00:00 +0200
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+---
+ 
+Many impressive first-ever-done scores, first places and more in this week’s article! 
+ 
+![](https://puu.sh/wA4Nj/d299c2b68d.png)
```

Comment:
Add to the repo under ``/wiki/shared/news`` following appropriate folder nomenclature for the filename